app/src/main/res/layout/activity_user_profile.xml

```diff
@@ -1,18 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
For whoever might merge this: Were the changes in this file made to make the action bar/ toolbar work? Because I have already done it in my branch and possibly other changes as well.